pyproject.toml

```diff
@@ -207,30 +207,33 @@ extend-exclude = [
 
 [tool.ruff.lint]
 extend-select = [
-    "ANN", # flake8-annotations
-    "B",   # flake8-bugbear
-    "C4",  # flake8-comprehensions
-    "FLY", # flynt
-    "I",   # isort
-    "ISC", # flake8-implicit-str-concat
-    "PGH", # pygrep-hooks
-    "PT",  # flake8-pytest-style
-    "PYI", # flake8-pyi
-    "RSE", # flake8-raise
-    "RET", # flake8-return
+    "ANN",  # flake8-annotations
+    "B",    # flake8-bugbear
+    "C4",   # flake8-comprehensions
+    "FLY",  # flynt
+    "FURB", # refurb
+    "I",    # isort
+    "ISC",  # flake8-implicit-str-concat
+    "PGH",  # pygrep-hooks
+    "PT",   # flake8-pytest-style
+    "PYI",  # flake8-pyi
+    "RSE",  # flake8-raise
+    "RET",  # flake8-return
     "RUF",
-    "TCH", # flake8-type-checking
-    "TRY", # tryceratops
-    "UP",  # pyupgrade
+    "TCH",  # flake8-type-checking
+    "TRY",  # tryceratops
+    "UP",   # pyupgrade
 ]
 ignore = [
     "ANN003",
     "ANN101",
     "ANN102",
     "ANN401",
-    "PT004",  # deprecated
-    "PT011",  # TODO: apply this rule
-    "PT012",  # TODO: apply this rule
+    "FURB118", # TODO: apply this rule
+    "FURB140", # TODO: apply this rule
+    "PT004",   # deprecated
+    "PT011",   # TODO: apply this rule
+    "PT012",   # TODO: apply this rule
```

Comment:
do we want to do these here?

Reply:
I'm just curious if these TODOs are for this PR are a later one.

Reply:
Let me apply these rules, I'm interested in your opinion.